feature-configs/demo/dpdk/dpdk-network.yaml

```diff
@@ -8,11 +8,11 @@ spec:
     {
       "type": "dhcp"
     }
-  networkNamespace: dpdk-testing
+  networkNamespace: dpdk
   resourceName: dpdknic
   vlan: 0
 ---
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: dpdk-testing
+  name: dpdk
```

Comment:
you are creating the namespace object in two different places (it also exist in build-config.yaml)
better to separate to it's own file, just as we've done in release-4.5 and master

Reply:
In the build config is an image stream I didn't find a namespace creation there.

But you are right I move it to a separate yaml file